templates/vm.tftpl

```diff
@@ -0,0 +1,167 @@
+%{ if vm_win_hostname != "" ~}
+module "win_vm" {
+  source                 = "qbeyond/windows-vm/azurerm"
+  version                = "4.1.0"
+  for_each               = local.vm_win
```

Comment:
I'm not up-to-date if we have made a decision on this yet so ignore if we already have, but my recommendation was to remove the module to locals indirection. If we just pipe all the locals 1 to 1 into this module we might as well just skip the locals to avoid "duplicate" code.

Reply:
Yes, it is true. I have changed to use the variable value directly in the module and descarted to use locals.